Code/RDGeneral/Dict.h

```diff
@@ -18,8 +18,8 @@
 #include <map>
 #include <string>
 #include <vector>
-#include <boost/any.hpp>
-#include <RDGeneral/Exceptions.h>
+#include "RDValue.h"
+#include "Exceptions.h"
```

Comment:
I'm curious about the motivation behind this change. Why do you prefer this style?